README.md

```diff
@@ -40,116 +73,57 @@ If you have any issues with OBT, please check out [troubleshooting guide](https:
 		--brand=<brand-name>       Build SCSS for a given brand. E.G. --brand=internal to build the component for the internal brand.
 		--debug                    Keep the test runner open to enable debugging in any browser.
 
-### Developing components locally
-
-Build and browse the demos (typically: <http://localhost:8080/demos/local/>),
-automatically re-build the component's demos and assets every time a file changes:
-
-	`obt dev`
+### Commands
```

Comment:
is this meant to be a subsection under usage? it looks from the ToC that Commands is a h2-level section

Reply:
good point, I'll make commands and api reference h2